mobile/library/swift/EnvoyCxxSwiftInterop/cxx_swift_interop.h

```diff
@@ -1,29 +1,53 @@
-#import "library/cc/bridge_utility.h"
-#import "library/cc/direct_response_testing.h"
-#import "library/cc/engine_builder.h"
-#import "library/common/data/utility.h"
-#import "library/common/extensions/filters/http/platform_bridge/c_types.h"
-#import "library/common/main_interface.h"
-#import "library/common/network/apple_platform_cert_verifier.h"
-#import "library/common/stats/utility.h"
+#include "library/cc/bridge_utility.h"
+#include "library/cc/direct_response_testing.h"
+#include "library/cc/engine_builder.h"
+#include "library/common/data/utility.h"
+#include "library/common/extensions/filters/http/platform_bridge/c_types.h"
+#include "library/common/main_interface.h"
+#include "library/common/network/apple_platform_cert_verifier.h"
+#include "library/common/stats/utility.h"
+
+// This file exists in order to expose headers for Envoy's C++ libraries
+// to Envoy Mobile's Swift implementation.
+// Further, Swift only supports interacting with a subset of C++ language features
+// so some types are renamed with the `using` keyword, and some features that
+// cannot be imported into Swift are abstracted via interfaces that are supported.
+// See this document on Swift's C++ interoperability status to learn more:
+// https://github.com/apple/swift/blob/swift-5.7.3-RELEASE/docs/CppInteroperability/CppInteroperabilityStatus.md
 
 namespace Envoy {
 namespace CxxSwift {
 
+using LogLevel = spdlog::level::level_enum;
 using StringVector = std::vector<std::string>;
 using StringPair = std::pair<std::string, std::string>;
 using StringPairVector = std::vector<StringPair>;
 using StringMap = absl::flat_hash_map<std::string, std::string>;
 using HeaderMatcherVector = std::vector<DirectResponseTesting::HeaderMatcher>;
 using BootstrapPtr = intptr_t;
```

Comment:
One trivial nit. For the other places where we "erase" a pointer into an int, we tend to name the type `envoy_something_t`. I wonder if we might want to name this `envoy_bootstrap_t` to be consistent? On the otherhand this is only for swift which uses a different convention, so maybe not?

Reply:
I’m leaning slightly towards keeping it PascalCase to follow Swift naming conventions.